tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -231,7 +233,9 @@ def _gen_kernel_library(
         types_with_unrolling_disabled = [],
         extra_args = [],
         test_tags = [],
-        test_size = "medium"):
+        test_size = "medium",
+        jit_i64_indexed_for_large_tensors_types = None,
+        output_partial_jit_types = None,):
```

Comment:
For consistency, maybe call it `output_jit_i64_indexed_for_large_tensors_types`

Reply:
done!

Reply:
For consistency, the two names should be `jit_i64_indexed_for_large_tensors_types` and `output_jit_i64_indexed_for_large_tensors_types`. 
This is analogous to `jit_types` and `output_jit_types`